astrocut/make_cube.py

```
@@ -346,3 +349,500 @@ def make_cube(self, file_list, cube_file="img-cube.fits", sector=None, max_memor
+
+                    for kwd in self.info_table.columns:  # Iterate over every keyword in the TICA FFI primary header
+                        if kwd == "FFI_FILE":
+                            self.info_table[kwd][i] = os.path.basename(fle)
+                            
+                        else:
+                            nulval = None
+                            if self.info_table[kwd].dtype.name == "int32":
+                                nulval = 0
+                            elif self.info_table[kwd].dtype.char == "S":  # hacky way to check if it's a string
+                                nulval = ""
+                            
+                            # NOTE: 
+                            # A header keyword ('COMMENT') in TICA returns a keyword 
+                            # value in the form of a _HeaderCommentaryCard instead of a STRING. 
+                            # This breaks the info table because the info table 
+                            # doesn't understand what a FITS header commentary card is. 
+                            # Adding a try/except is a way to catch when these instances happen 
+                            # and turn the keyword value from a _HeaderCommentaryCard to a
+                            # string, which is what it's meant to be in the info table. 
+                            #
+                            # TO-DO: Find a more elegant way to handle these stray 
+                            # _HeaderCommentaryCards. 
+                            
+                            try:
+                                self.info_table[kwd][i] = ffi_data[0].header.get(kwd, nulval)
+                            except ValueError:
+                                kwd_val = ffi_data[0].header.get(kwd)
+                                if isinstance(kwd_val, fits.header._HeaderCommentaryCards):
+                                    self.info_table[kwd][i] = str(kwd_val)
```

if we raise a valueerror here and it's not an instance of _headerCOmmentaryCards, we just silently ignore the error.

We should add a `raise` here at the end

```suggestion
                            except ValueError:
                                kwd_val = ffi_data[0].header.get(kwd)
                                if isinstance(kwd_val, fits.header._HeaderCommentaryCards):
                                    self.info_table[kwd][i] = str(kwd_val)
                                else:
                                    raise
```